src/components/navbar.js

```diff
@@ -60,6 +60,20 @@ function Navbar(props) {
             </span>
           </Link>
 
+          <Link
+            to="/deepdive"
+            onClick={() => {
+              setView("DeepDive");
+            }}
+          >
+            <span
+              className={`fadeInUp ${view === "Network Map" ? "focused" : ""}`}
```

Comment:
It should be **"DeepDive"** instead of **"Network Map"**.
This line is used to highlight the current selection.

Also if possible, at line 29 please change the **"Network Map"** to **"Clusters"**

Reply:
Thanks @KrishnaPravin. Updated!